src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx

```diff
@@ -128,12 +128,16 @@ export default function DirectSupportedCampsUI({
                           <div className={styles.btndiv}>
                             {" "}
                             <span className="count">{tag.id}. </span>
-                            <Link href={tag.camp_link}>
-                              <a className={styles.Bluecolor}>
-                                {" "}
-                                {tag.camp_name}
-                              </a>
-                            </Link>
+                            <a
```

Comment:
We need to use Link here

Reply:
@akuks Link is causing an issue in the drag-and-drop functionality which is why I have removed the link here

Reply:
@akuks Link is causing an issue in drag and drop functionality which is why I have removed the link here